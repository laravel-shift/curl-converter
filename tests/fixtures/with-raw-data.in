curl 'https://api.com' --header 'Accept: application/json' --header 'Content-Type: application/json' --data-raw '{
    "messages": [
        "a",
        "b",
        "c"
    ]
}'
