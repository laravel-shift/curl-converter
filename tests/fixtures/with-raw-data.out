Http::withBody('{
    "messages": [
        "a",
        "b",
        "c"
    ]
}')
    ->acceptJson()
    ->post('https://api.com');
